loleaflet/Makefile.am

```diff
@@ -363,7 +363,8 @@ LOLEAFLET_JS_WEBORDER =\
 
 if ENABLE_FEEDBACK
 LOLEAFLET_JS_WEBORDER +=\
-	src/control/IFrameDialog.js
+	src/control/IFrameDialog.js \
+	src/map/handler/Map.Feedback.js
 endif
 
 LOLEAFLET_JS_WEBORDER +=\
```

Comment:
Solved conflits

Reply:
keep it the LOLEAFLET_JS_LST,  only change
LOLEAFLET_JS_WEBORDER = $(filter %,$(LOLEAFLET_JS_LST))

otherwise probably unit test will fail

Reply:
```bash
++<<<<<<< HEAD
 +# the ordering of this is delicate to get JS initialization right,
 +# this list includes both Javascript and Typescript generated files
 +# in a rather precise order.
 +LOLEAFLET_JS_WEBORDER =\
Sorry I didn't understand the sentence "keep it the LOLEAFLET_JS_LST, only change LOLEAFLET_JS_WEBORDER ". Should I keep `LOLEAFLET_JS_WEBORDER` (master) or replace it with `LOLEAFLET_JS_LST`(cherry-pick)

these are the conflicts:

++=======
+ LOLEAFLET_JS_LST =\
++>>>>>>> ac6a4c3f5... loleaflet: add initial iframe dialog
```

```bash
++<<<<<<< HEAD
 +# the ordering of this is delicate to get JS initialization right,
 +# this list includes both Javascript and Typescript generated files
 +# in a rather precise order.
 +LOLEAFLET_JS_WEBORDER =\
++=======
+ LOLEAFLET_JS_LST =\
++>>>>>>> ac6a4c3f5... loleaflet: add initial iframe dialog

```

Reply:
pick the LOLEAFLET_JS_LST